js/remote/index.js

```diff
@@ -85,7 +107,7 @@ function onClientDisconnect() {
     console.log('client disconnected');
     /*
      * TODO:
-     * should this interval be cleared here when the client disconnects from the bot
+     * should this interval be cleared here when the client disconnects
      * or explicitly as a command from the surface when it disconnects or both?
```

Comment:
I think the most robust practice would be both, but I'm not sure.

Reply:
This is an old comment, it actually does do both now (oops).